source/dagger/gameplay/platformer/platformer_controller.cpp

```diff
@@ -40,6 +40,36 @@ void PlatformerControllerSystem::Run()
                 sprite_.scale.x = run;
                 sprite_.position.x += char_.speed * sprite_.scale.x * Engine::DeltaTime();
             }
+            Float32 jump = 0;
+            if (!char_.jumping)
+                jump = input_.values.at("jump");
+            if (jump)
+            {
+                char_.jumping = true;
+            }
+
+            if (char_.jumping)
+            {
+                int maxHight = 60;
```

Comment:
nit pick: `maxHeight` misspelled

Reply:
I'd take this `maxHeight` into data, for now maybe loaded through your ini file?

Reply:
I'll put it in data, it does make much more sense. Should I make another pull request when I fix that, or just merge it, since it's a minor change?

Reply:
Nah, keep it in this.